hyperjumptech/monika-config-generator
# added: add probes and notifications on the sidebar

### What feature/issue does this PR add
Add probes and notifications list on the sidebar. Resolve #95.

### How did you implement / how did you fix it
- Show the names of the probes under Probe menu on the sidebar.
- Show the names of the notification types under the Notifications menu on the sidebar.
- When a probe on the sidebar is clicked, scroll to that probe.
- When a notification on the sidebar is clicked, scroll to that notification.
- When user scrolls down the page, the sidebar should stay visible and sticky to the top.

### How to test
1. Run `npm run dev`
2. Open `http://localhost:3000`
3. Choose `I have a configuration file` and click `Next`
4. Upload Monika config file with many probes and notifications
5. See the sidebar, click probe and notification on the sidebar
